src/codegen/revert.rs

```diff
@@ -35,18 +36,24 @@ pub enum SolidityError {
     /// Reverts with "empty error data"; stems from `revert()` or `require()` without string arguments.
     Empty,
     /// The `Error(string)` selector
-    String(Expression),
+    Error(Expression),
     /// The `Panic(uint256)` selector
     Panic(PanicCode),
+    /// User defined errors
+    Custom {
+        error_no: usize,
+        expr: Vec<Expression>,
```

Comment:
`exprs`? We usually use `expr` for singular.

Reply:
Changed that